build/cloudbuild-tf-apply.yaml

```diff
@@ -22,8 +22,11 @@ steps:
   args:
   - -c
   - |
-    echo "Setting up gcloud for impersonation"
-    gcloud config set auth/impersonate_service_account ${_TF_SA_EMAIL}
+   tf_sa_email=${_TF_SA_EMAIL}
```

Comment:
Also it looks like this substitution won't be available for app infra triggers?

Reply:
Correct, that was my reason for wrapping in a if statement